R/utils.R

```diff
@@ -129,3 +129,32 @@ find_midpoint.Histogram <- function(x){
 find_midpoint.GenomicHistogram <- function(x){
   (x$consecutive_start + x$consecutive_end)/2
 }
+
+#' Loads base config file and overwrites with an optional additional file
+#'
+#' @param file optional file path to user config script
+#'
+#' @return list of yaml config
```

Comment:
Is this a function that users might potentially need or is it more paper-analysis specific?